clients/java/zts/core/src/main/java/com/yahoo/athenz/zts/ZTSClient.java

```diff
@@ -1505,7 +1559,7 @@ public void run() {
             boolean svcTokenRefresh = false;
             for (PrefetchTokenScheduledItem item : PREFETCH_SCHEDULED_ITEMS) {
 
-                // see if item expires within next two minutes
+                // see if item requires refresh
```

Comment:
The current logic refreshes the token once it gets to half time before the expiration.
I didn't had to change this logic, I just had to make sure that notification will be sent in case of failure